make/release.mk

```
@@ -22,5 +22,5 @@ release.s3:
 ifeq ($(CATALOG_APPLICATIONS_VERSION),"")
 	$(info CATALOG_APPLICATIONS_VERSION should be set to the version which is part of the s3 file path)
 else
-	aws s3 cp --no-progress --acl bucket-owner-full-control $(IMAGE_TAR_FILE) s3://$(RELEASE_S3_BUCKET)/dkp/$(CATALOG_APPLICATIONS_VERSION)/catalog_applications_image_bundle_$(CATALOG_APPLICATIONS_VERSION)_linux_amd64.tar.gz
+	aws s3 cp --no-progress --acl bucket-owner-full-control $(IMAGE_TAR_FILE) s3://$(RELEASE_S3_BUCKET)/dkp/$(CATALOG_APPLICATIONS_VERSION)/catalog-applications-image-bundle-$(CATALOG_APPLICATIONS_VERSION).tar.gz
```

maybe add `dkp-` prefix to keep naming same as repo ?
```suggestion
	aws s3 cp --no-progress --acl bucket-owner-full-control $(IMAGE_TAR_FILE) s3://$(RELEASE_S3_BUCKET)/dkp/$(CATALOG_APPLICATIONS_VERSION)/dkp-catalog-applications-image-bundle-$(CATALOG_APPLICATIONS_VERSION).tar.gz
```

---

then i'll have to go back and update https://github.com/mesosphere/kommander/pull/1605/files#diff-70acc2689fb0b36f116699b4dd558c7361f4cc683cd8a4dd6f1e559a76e9a674R159 😅

---

in all other filenames i have seen `file-name-with-dash-but-underscore-before-version_version_with_dashes`.
i dont care which one but it would be good to have some consistent pattern :see_no_evil:

---

good idea! done

---

> in all other filenames i have seen `file-name-with-dash-but-underscore-before-version_version_with_underscores`. i dont care which one but it would be good to have some consistent pattern 🙈

yeah this PR https://github.com/mesosphere/kommander/pull/1605/files#diff-70acc2689fb0b36f116699b4dd558c7361f4cc683cd8a4dd6f1e559a76e9a674R159 moved to using all dashes with no underscores